src/tiled/mainwindow.cpp

```diff
@@ -380,6 +380,9 @@ MainWindow::MainWindow(QWidget *parent, Qt::WindowFlags flags)
     mViewsAndToolbarsMenu = new QMenu(this);
     mViewsAndToolbarsAction = new QAction(tr("Views and Toolbars"), this);
     mViewsAndToolbarsAction->setMenu(mViewsAndToolbarsMenu);
+
+    mResetDefaultLayout = new QAction(tr("Reset To Default Layout"),this);
```

Comment:
"to" should be in lowercase (English title casing rules).

Coding style: please put a space after all commas.